src/TraceEvent/TraceLog.cs

```diff
@@ -30,6 +30,7 @@
 using System.Text.RegularExpressions;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.Diagnostics.Tracing.Parsers.ProcessMetadataEventSource;
```

Comment:
I know that in one of your other PRs, you had done some work to expose process name inside of the `EventPipe` source.  Do you think that would be worth plumbing in here as well?

Reply:
Done.